src/Grok.Net/Grok.cs

```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
+using System.Text.RegularExpressions;
```

Comment:
the grok since v2.0.0 doesn't use the `System.Text.RegularExpressions` anymore, it uses PCRE. https://github.com/Marusyk/grok.net/releases/tag/v2.0.0

Reply:
Ok sounds good. I have updated my code.